lib/src/Picker/Picker.tsx

```diff
@@ -41,11 +41,11 @@ const muiComponentConfig = { name: 'MuiPickersBasePicker' };
 
 export const useStyles = makeStyles(
   {
-    container: {
+    root: {
       display: 'flex',
       flexDirection: 'column',
     },
-    containerLandscape: {
+    rootLandscape: {
```

Comment:
```suggestion
    landscape: {
```

as we already do: https://github.com/mui-org/material-ui/blob/1691d498b3c7fc8c7f4337948e7ad7c2947f66a2/packages/material-ui/src/Stepper/Stepper.js#L20?